code/ui/blocks/src/blocks/Meta.tsx

```diff
@@ -5,13 +5,13 @@ import type { BaseAnnotations, ModuleExports } from '@storybook/types';
 import { Anchor } from './Anchor';
 import { DocsContext } from './DocsContext';
 
-type MetaProps = BaseAnnotations & { of?: ModuleExports };
+type MetaProps = BaseAnnotations & { of?: ModuleExports; title?: string };
 
 /**
  * This component is used to declare component metadata in docs
  * and gets transformed into a default export underneath the hood.
  */
-export const Meta: FC<MetaProps> = ({ of }) => {
+export const Meta: FC<MetaProps> = ({ of, title }) => {
```

Comment:
I don't think you need to destructure the `title` out of props here as it's never being used.

Reply:
@MWhite-22 thank u for responding. I have updated this please can you consider this PR for merging so that i can get  my first PR merged.